pkg/apis/apps/v1alpha1/broadcastjob_types.go

```diff
@@ -28,7 +28,7 @@ type BroadcastJobSpec struct {
 	// be less than this number when the work left to do is less than max parallelism.
 	// Not setting this value means no limit.
 	// +optional
-	Parallelism *int32 `json:"parallelism,omitempty" protobuf:"varint,1,opt,name=parallelism"`
+	Parallelism *string `json:"parallelism,omitempty" protobuf:"varint,1,opt,name=parallelism"`
```

Comment:
Why not Parallelism intstr.IntOrString?  I think it is ok to break backward compatibility for now as long as we document it well in change log.